bin/yaml/swift.yaml

```diff
@@ -52,9 +57,9 @@ compilers:
       type: restQueryTarballs
       install_always: true
       toolchain: development
-      url: https://download.swift.org/{{toolchain}}/ubuntu2004/latest-build.yml
+      url: https://download.swift.org/{{toolchain}}/ubuntu2404/latest-build.yml
       query: |
-        'https://download.swift.org/{{toolchain}}/ubuntu2004' \
+        'https://download.swift.org/{{toolchain}}/ubuntu2404' \
```

Comment:
i know little about what these Ubuntu versions mean – is there any risk of the payload being different in a way that would be a problem for the existing infrastructure? i tested this change locally and the download appeared to work fine...

Reply:
will there be no 2204 links either? we don't support 24 yet and thusit will break the compilers if 24 is used. We're running on 22.04 currently

Reply:
i believe there will still be 2204 links. i will update to those instead. thanks!

Reply:
yes